packages/interface/src/components/interface-skeleton/index.js

```diff
@@ -24,72 +24,84 @@ function useHTMLClass( className ) {
 	}, [ className ] );
 }
 
-function EditorSkeleton( {
+function InterfaceSkeleton( {
 	footer,
 	header,
 	sidebar,
 	content,
-	publish,
+	actions,
+	labels,
 	className,
 } ) {
-	useHTMLClass( 'block-editor-editor-skeleton__html-container' );
+	useHTMLClass( 'interface-interface-skeleton__html-container' );
+
+	const defaultLabels = {
+		header: __( 'Editor header' ),
+		body: __( 'Editor content' ),
+		sidebar: __( 'Editor settings' ),
+		actions: __( 'Editor publish' ),
+		footer: __( 'Editor footer' ),
+	};
```

Comment:
Do you think we should drop the "Editor" from the default labels?

Reply:
I also feel it works better for most UIs without "Editor" (like widgets)

Reply:
Yes, that sounds fine to me. 👍

Reply:
Updated in 2e1d036ae02a7f5eceed238059815bb05e750284.

Reply:
So this changes the labels for the widgets and more importantly the "Post editor". For me it's still good names and I like the fact that they are the same across pages. Let's keep them consistent for now and if we have feedback suggesting to go specific for some of these screens, we'll adapt.

Reply:
> Do you think we should drop the "Editor" from the default labels?

When used to label the ARIA landmark regions in the context of the WP admin, these labels need to differentiate the Editor landmarks from the _other_ landmarks in the WP UI. This is the reason why they used the word "Editor" at the beginning. For some history, see https://github.com/WordPress/gutenberg/pull/7940, https://github.com/WordPress/gutenberg/issues/7938, and https://github.com/WordPress/gutenberg/pull/2380

Also, the ARIA landmarks labels need to describe the purpose of the content within the region. Labels like "Header", "Footer", and "Left Sidebar" (added in https://github.com/WordPress/gutenberg/pull/20951) are arguably og any usefulness for assistive technology users.

If these labels are going to be used also for other purposes, then we'll need to provide specific labels for the ARIA landmarks.

Worth also noting the ARIA regions labels usage is still documented in the regions Readme:
> For better accessibility, these elements must be properly labelled to briefly describe the purpose of the content in the region. For more details, see "ARIA landmarks" in the [WAI-ARIA specification](https://www.w3.org/TR/wai-aria/).

Will create a new issue.